src/main/resources/templates/index.html

```diff
@@ -56,6 +41,24 @@
         integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p"
         crossorigin="anonymous"></script>
 
+
+<p th:text="${item}" />
```

Comment:
All of this should be inside the `div class=row` on line 35. Also, please make use of the bootstrap form classes